sim/gui.py

```diff
@@ -88,21 +139,21 @@ def draw_robot(cls, screen: pygame.surface.Surface, robot_pos: tuple[int, int],
         pygame.draw.line(screen, cls.robot_second_color, robot_pos, heading_point, 1)
 
     @classmethod
-    def draw_text(cls, screen: pygame.surface.Surface, text: str, size: int, center: tuple[int, int], color='white'):
+    def draw_text(cls, screen: pygame.surface.Surface, text: str, size: int, center: Position, color='white'):
         """Draw text on screen.
 
         Args:
             screen (pygame.surface.Surface): screen to draw on.
             text (str): text to draw.
             size (int): size of text.
-            center (tuple[int, int]): center of text on screen (x, y).
+            center (Position): center of text on screen (x, y).
             color (str, optional): text's color. Defaults to 'white'.
         """
         font = pygame.font.Font(pygame.font.get_default_font(), size)
         text = font.render(text, True, color)
-        textRect = text.get_rect()
-        textRect.center = center
-        screen.blit(text, textRect)
+        text_rect = text.get_rect()
+        text_rect.center = (center[0], center[1])
```

Comment:
Why not just

```suggestion
        text_rect.center = center
```

or

```suggestion
        text_rect.center = tuple(center)
```
?